### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t2080ba

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by jbuck** [comment]
Brett, the first thing we do tomorrow is figure out why this doesn't work for you.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e3cf32d02efd659/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/e3cf32d02efd659/output.txt

Total script time: 0.27 mins

File: css/header.less [hunk 1]
```diff
@@ -19,52 +19,193 @@
...
-  position: relative;
-  float: left;
-  height: @HEADER_SPACING;
+  position: absolute;
+  top: 0;
+  left: 30px;
+  height: 100px;
+  width: @_logoWidth;
+  background: url( '../resources/logo-words.png' ) no-repeat;
+  z-index: 100000;
```
Comment (secretrobotron) [inline css/header.less:41]: Does this absolutely need its own z-index?

File: src/ui/header.js [hunk 2]
```diff
@@ -15,6 +15,10 @@ define( [ "dialog/dialog", "util/lang", "text!layouts/header.html", "ui/widget/t
         _authButton = _rootElement.querySelector( ".butter-login-btn" ),
         _projectTitle = _rootElement.querySelector( ".butter-project-title" ),
         _projectName = _projectTitle.querySelector( ".butter-project-name" ),
+        _nameDropDown = _buttonGroup.querySelector( "ul" ),
+        _logoutBtn = _nameDropDown.querySelector( ".butter-logout-btn"),
```
Comment (secretrobotron) [inline src/ui/header.js:19]: Spacing.

File: src/ui/header.js [hunk 3]
```diff
@@ -15,6 +15,10 @@ define( [ "dialog/dialog", "util/lang", "text!layouts/header.html", "ui/widget/t
...
+        _myProjectsButton = _nameDropDown.querySelector( ".butter-my-projects-btn"),
```
Comment (secretrobotron) [inline src/ui/header.js:20]: Spacing.

File: src/ui/header.js [hunk 4]
```diff
@@ -15,6 +15,10 @@ define( [ "dialog/dialog", "util/lang", "text!layouts/header.html", "ui/widget/t
...
+        _tabzilla = _rootElement.querySelector( "#tabzilla "),
```
Comment (secretrobotron) [inline src/ui/header.js:21]: Probably spacing.

File: css/header.less [hunk 5]
```diff
@@ -19,52 +19,193 @@
...
+    &:last-child {
+      border-right: none;
+      box-shadow: none;
+    }
+  }
+}
+
+#tabzilla {
+  display: block;
+  background: url( "../resources/tabzilla.png" );
```
Comment (secretrobotron) [inline css/header.less:82]: Don't need this png, right? tabzilla's css already includes one that matches the rest of the tabzilla design. Don't want to have our own if theirs changes slightly, right?

**Pull request closed without merging**